### ISSUE TITLE
Submit button on Report a problem button has unnecessary title

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#97]
URL: https://hcdev.openplus.ca/fr/avis-rappel/test-rappel-salades-chou-frise-marque-curation-foods-et-salades-hachees-pretes-servir
User Agent: Chrome
Language:  both EN and FR.

Issue Description:
The “title” attribute of the “Submit” button says “Search”. It is not correct. In this case, the “title” attribute is not necessary and should be removed.          
	
ENGLISH: 
<button class="webform-button--submit button button--primary js-form-submit form-submit btn-primary btn" data-drupal-selector="edit-submit" type="submit" id="edit-submit--2" name="op" value="Submit" **title="Search"**>Submit</button>

FRENCH: 
<button class="webform-button--submit button button--primary js-form-submit form-submit btn-primary btn" data-drupal-selector="edit-submit" type="submit" id="edit-submit--2" name="op" value="Soumettre" **title="Recherche"**>Soumettre</button>

Steps to reproduce: 
1. Navigate to the URL, expand the Report a problem button
2. View code for the Submit button.
   OR 
hover your mouse over the button, the tooltip will display the "title" attribute. 

Expected results:
Remove the title attribute it is unnecessary as it provides no additional information to users.

**Comment by echouin** [comment]
@mmiron-HC OP has indicated that this is now fixed :)

**Comment by ndolter** [comment]
Validated in both French and English that the tooltip has been removed from the Submit button.
